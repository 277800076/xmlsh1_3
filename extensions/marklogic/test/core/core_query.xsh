# test basic query 
. ../init

xecho <[ 
   <foo>
      <elem name="a">data a</elem>
      <elem name="b">data b</elem>
      <elem name="c">data c</elem>
   </foo>
]> | ml:put -uri /test_query.xml 

ml:query -q <{{
   declare variable $name external ;
   doc("/test_query.xml")//elem[@name=$name]
 }}> -v name "b" 
 ml:del /test_query.xml
 
