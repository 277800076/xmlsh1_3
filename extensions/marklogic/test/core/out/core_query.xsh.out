<elem name="b">data b</elem>
